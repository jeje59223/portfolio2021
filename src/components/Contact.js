import React from "react";
import "../styles/contact.css";

const Contact = () => {
    return (
        <section id="contact" className="bg-light py-5">
            <div className="container">
                <div className="row">
                    <div className="col-lg-12">
                        <h3>Contactez-moi</h3>
                        <p className="lead">
                        Une interrogation ? Un conseils ? Ou juste envie d'échanger avec moi ? N'hésitez pas à me contacter !
                        </p>
                        <form>
                            <div className="input-group mb-3">
                                <span className="input-group-text" id="basic-addon1"><i className="fas fa-user"></i></span>
                                <input type="text" className="form-control" placeholder="Username" aria-label="Username" aria-describedby="basic-addon1"/>
                            </div>
                            <div className="input-group mb-3">
                                <span className="input-group-text" id="basic-addon1"><i className="fas fa-envelope"></i></span>
                                <input type="mail" className="form-control" placeholder="Email" aria-label="Username" aria-describedby="basic-addon1"/>
                            </div>
                            <div className="input-group">
                                <span className="input-group-text"><i className="fas fa-pencil-alt"></i></span>
                                <textarea rows="5" className="form-control" aria-label="With textarea"></textarea>
                            </div>
                            <input type="submit" value="Envoyer" className="btn btn-secondary btn-block btn-lg mt-3"/>
                        </form>
                    </div>
                </div>
            </div>
        </section>
    )
}

export default Contact;