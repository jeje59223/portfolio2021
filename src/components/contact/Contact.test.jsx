import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Contact from './Contact';

configure({ adapter: new Adapter() });

describe('<Contact>', () => {
  it('should render Contact Form', () => {
    const wrapper = shallow(<Contact />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});
