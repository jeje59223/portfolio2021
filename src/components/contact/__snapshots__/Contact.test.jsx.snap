// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact> should render Contact Form 1`] = `
<section
  className="bg-light py-5"
  id="contact"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12"
      >
        <h3>
          Contactez-moi
        </h3>
        <p
          className="lead"
        >
          Une interrogation ? Un conseils ? Ou juste envie d'échanger avec moi ? N'hésitez pas à me contacter !
        </p>
        <form>
          <div
            className="input-group mb-3"
          >
            <span
              className="input-group-text"
              id="basic-addon1"
            >
              <i
                className="fas fa-user"
              />
            </span>
            <input
              aria-describedby="basic-addon1"
              aria-label="Username"
              className="form-control"
              placeholder="Username"
              type="text"
            />
          </div>
          <div
            className="input-group mb-3"
          >
            <span
              className="input-group-text"
              id="basic-addon1"
            >
              <i
                className="fas fa-envelope"
              />
            </span>
            <input
              aria-describedby="basic-addon1"
              aria-label="Username"
              className="form-control"
              placeholder="Email"
              type="mail"
            />
          </div>
          <div
            className="input-group"
          >
            <span
              className="input-group-text"
            >
              <i
                className="fas fa-pencil-alt"
              />
            </span>
            <textarea
              aria-label="With textarea"
              className="form-control"
              rows="5"
            />
          </div>
          <input
            className="btn btn-secondary btn-block btn-lg mt-3"
            type="submit"
            value="Envoyer"
          />
        </form>
      </div>
    </div>
  </div>
</section>
`;
